src/Models/Database.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+namespace Cms\Models;
+
 require "dbconfig.php";
```

Comment:
Create a separate issue for including the database credentials... .. make it more efficient..

Reply:
#41 created a new issue for autoloading the dbconfig file.